code-challenges.test.js

```diff
@@ -38,36 +95,75 @@ const studyMinutesWeek1 = {
+
+
+// // b) Create the function that makes the test pass.
+//Create a function and declare a variable that takes in an object
+
+const sortedLeast = (object) => {
+  const studyMinutesWeek= (studyMinutesWeek1.value)
+  studyMinutesWeek1.sort((a,b)=> a-b)
+  return sortedLeast
 }
-// Expected output: [10, 15, 20, 45, 60, 65, 100]
+// The value is required so we create a variable to store the objects value
+//usingh .sort method, we are able to sort the arrValue numbers from least to greatest.
 
-// b) Create the function that makes the test pass.
+//ERRORS:
+//TypeError: Cannot read properties of undefined (reading 'value')
 
-// Pseudo code:
+// // Pseudo code:
 
-// --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
+// // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
+
+// // a) Create a test with expect statements for each of the variables provided.
+
+describe("inOrder", () => {
+  it("returns an array of the accumulating sum", () => {
+    expect(inOrder()).toEqual([100, 83, 60, 51])
+    expect(inOrder()).toEqual([250, 161, 261, 165])
+    expect(inOrder()).toEqual([])
+  })
+})
+
+
+// // const accountTransactions1 = [100, -17, -23, -9]
+// // // Expected output: [100, 83, 60, 51]
+
+// // const accountTransactions2 = [250, -89, 100, -96]
+// // // Expected output: [250, 161, 261, 165]
+
+// // const accountTransactions3 = []
+// // Expected output: []
+
+// // b) Create the function that makes the test pass.
+// //Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
-// a) Create a test with expect statements for each of the variables provided.
 
-const accountTransactions1 = [100, -17, -23, -9]
-// Expected output: [100, 83, 60, 51]
 
-const accountTransactions2 = [250, -89, 100, -96]
-// Expected output: [250, 161, 261, 165]
+// // Pseudo code:
+// //Input: Array of numbers
+// //Output: Array of numbers
+// //Create  function that takes in array of numbers as the parameter
+// //.map() method to iterate over array and adds accumulating sum using appropriate operators
+// Values will need to be stored, create a new variable
 
-const accountTransactions3 = []
-// Expected output: []
 
-// b) Create the function that makes the test pass.
+const inOrder = (arr) => {
+const addSum = accountTransactions1.map()
+//I was unsure of how to get the accumulating sum. I know that a for loop would be able to loops around until the condition of adding is met but not sure of the syntax. 
+//for (let i = 1; i + num) // or something similar is what I was thinking but my tests failed. I want to add [100, -17, -23, -9] for an output of [100, 83, 60, 51].
 
-// Pseudo code:
+//Each value 0 index + 1 index... and getting it to appear is what is most confusing.
```

Comment:
That's okay! You're on the right track thinking of using the index and adding the value to the value at the next index. Definitely work through this one with your mentor!